shared/engine/index-impl.tsx

```diff
@@ -14,6 +14,13 @@ import {CustomResponseIncomingCallMapType, IncomingCallMapType} from '.'
 import {SessionID, SessionIDKey, WaitingHandlerType, MethodKey} from './types'
 import {TypedState, Dispatch} from '../util/container'
 
+// delay incoming to stop react from queueing too many setState calls and stopping rendering
+// only while debugging for now
+const DEFER_INCOMING_DURING_DEBUG = __DEV__ && true
```

Comment:
- [ ] set to false before committing

This is to solve that issue where if you hang at a breakpoint while rendering react will die